spec/linter-tidy-spec.js

```diff
@@ -4,8 +4,27 @@ import * as path from 'path';
 
 const lint = require(path.join('..', 'lib', 'main.coffee')).provideLinter().lint;
 
+// TODO: Implement variables and functions related to the main linter package.
```

Comment:
There isn't really a need for this, the providers interact with the `linter` package (or anything else consuming the API, like Nuclide!) through Atom's services API, so the only way it wouldn't be working properly is if that was broken... and testing that is already done within Atom.